### PR TITLE
Accessibility Guidelines

### LINKED ISSUE #161: Ensure web application is accessible!
Some tips for making web applications as accessible as possible, for differently-sighted users (and also just making apps better in general):
* Using HTML
    * Provide alt tags for images
    * Pay attention to the tabindex
    * Headers (h1-h6): actually use them! Organize documents like trees, and don’t skip levels. 
    * Mark things as headers, articles, figures, footers. (Look here: https://www.w3schools.com/tags/)
* Using ARIA to actually define the roles of different tags (e.g. role=“button”): https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA

Web content disability guidelines: https://www.w3.org/TR/WCAG21/

### PR DESCRIPTION
**Pull request by bourque**
This PR includes various changes to the web app templates to conform to accessibility guidelines (https://innerspace.stsci.edu/pages/viewpage.action?spaceKey=A5C&title=Webpage+Checklist)

Closes #161

### CODE REVIEW

**Comment by bourque** [comment]
@laurenmarietta @catherine-martlin Could we all just give this PR one more look-through/review before we merge?  I will also do this, and test everything out locally.

File: style_guide/accessibility_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+Guidelines for Accessible Development
```
Comment (bourque) [inline style_guide/accessibility_guidelines.md]: This is an awesome resource!

**Comment by bourque** [comment]
For the record, both the code changes and the aesthetic changes look good to me!

**Comment by bourque** [comment]
Since we are approaching our TIPS talk, I am going to go ahead and merge this

**Comment by catherine-martlin** [comment]
Go ahead! It looks good to me!

**Pull request merged**